src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './App.css';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import Navbar from './components/Navbar'
+import Home from './pages/HomePage'
+
+
+function App() {
+  return (
+    <Router>
```

Comment:
What is the point of this router? Seems like an unnecessary use of a library. It is not being used to do anything.